app/views/contacts/show.html.slim

```diff
@@ -0,0 +1,50 @@
+.row
+  .col-xs-12
+    h1= t_title(:show)
+
+.row
+  .col-xs-12
+    dl.dl-horizontal
+      - if kind == 'person'
+        dt= t_attr(:first_name)
+        dd= @contact.first_name
+
+        dt= t_attr(:last_name)
+        dd= @contact.last_name
+
+        dt= t_attr(:title)
+        dd= @contact.title
+
+        dt= t_attr(:gender)
+        dd= @contact.gender
+
+        dt= t_attr(:birthday)
+        dd= @contact.birthday
+
+        dt= t_attr(:birthplace)
+        dd= @contact.birthplace
+
+      dt= t_attr(:organisation)
+      dd= @contact.organisation
+
+      dt= t_attr(:org_role)
+      dd= @contact.org_role
+
+- if kind == 'person'
```

Comment:
Those fields are not part of your story, so drop them. @chrysalag  has them allready in her story for scaffolding volunteers. Also those fields are not part of contact informations. Later on when we have an address book, they wont have those fields too.

If we need them unified, we still can do an additional story. But that is not part of your story.
Oh and you don't decide what gets removed and what not, that is not your role.